docs/rules/no-await-sync-events.md

```diff
@@ -45,6 +53,12 @@ const bar = () => {
   userEvent.tab();
   // ...
 };
+
+const baz = () => {
+  // ...
+  await userEvent.type(textInput, 'abc', {delay: 1000});
```

Comment:
Can you add an extra correct example for `userEvent.type` without await and delay please?

Reply:
updated